megamek/src/megamek/common/Targetable.java

```diff
@@ -151,4 +151,17 @@ default boolean tracksHeat() {
     
     @Override
     int hashCode();
+    
+    /**
+     * Utility function used to safely tell whether two Targetables are in the same hex.
+     * Does not throw exceptions in case of nulls.
+     */
+    public static boolean atSamePosition(Targetable first, Targetable second) {
```

Comment:
Nullable annotate both first and second

Reply:
Further, for the name... isAtSamePosition would probably be a better name than atSamePosition.